utils/build/docker/python/parametric/apm_test_client/server.py

```diff
@@ -730,6 +737,383 @@ def otel_set_attributes(args: OtelSetAttributesArgs):
+    )
+
+    instrument_key = ",".join(
+        [args.meter_name, args.name.strip().lower(), "observable_counter", args.unit, args.description]
+    )
+    otel_meter_instruments[instrument_key] = observable_counter
+    return OtelCreateAsynchronousCounterReturn()
+
+
+class OtelCreateAsynchronousUpDownCounterArgs(BaseModel):
+    meter_name: str
+    name: str
+    description: str
+    unit: str
+    value: float
+    attributes: dict
+
+
+class OtelCreateAsynchronousUpDownCounterReturn(BaseModel):
+    pass
+
+
+def create_constant_observable_updowncounter_func(value: float, attributes: dict):
+    def observable_updowncounter_func(options: CallbackOptions):
+        yield Observation(value, attributes)
+
+    return observable_updowncounter_func
+
+
+@app.post("/metrics/otel/create_asynchronous_updowncounter")
+def otel_create_asynchronous_updowncounter(args: OtelCreateAsynchronousUpDownCounterArgs):
+    if args.meter_name not in otel_meters:
+        raise ValueError(f"Meter name {args.meter_name} not found in registered meters {otel_meters.keys()}")
+
+    meter = otel_meters[args.meter_name]
+    observable_updowncounter = meter.create_observable_up_down_counter(
+        args.name,
+        [create_constant_observable_updowncounter_func(args.value, args.attributes)],
+        args.unit,
+        args.description,
+    )
+
+    instrument_key = ",".join(
+        [args.meter_name, args.name.strip().lower(), "observable_updowncounter", args.unit, args.description]
+    )
+    otel_meter_instruments[instrument_key] = observable_updowncounter
+    return OtelCreateAsynchronousUpDownCounterReturn()
+
+
+class OtelCreateAsynchronousGaugeArgs(BaseModel):
+    meter_name: str
+    name: str
+    description: str
+    unit: str
+    value: float
+    attributes: dict
+
+
+class OtelCreateAsynchronousGaugeReturn(BaseModel):
+    pass
+
+
+def create_constant_observable_gauge_func(value: float, attributes: dict):
+    def observable_gauge_func(options: CallbackOptions):
+        yield Observation(value, attributes)
+
+    return observable_gauge_func
+
+
+@app.post("/metrics/otel/create_asynchronous_gauge")
+def otel_create_asynchronous_gauge(args: OtelCreateAsynchronousGaugeArgs):
+    if args.meter_name not in otel_meters:
+        raise ValueError(f"Meter name {args.meter_name} not found in registered meters {otel_meters.keys()}")
+
+    meter = otel_meters[args.meter_name]
+    observable_gauge = meter.create_observable_gauge(
+        args.name, [create_constant_observable_gauge_func(args.value, args.attributes)], args.unit, args.description
+    )
+
+    instrument_key = ",".join(
+        [args.meter_name, args.name.strip().lower(), "observable_gauge", args.unit, args.description]
+    )
+    otel_meter_instruments[instrument_key] = observable_gauge
+    return OtelCreateAsynchronousGaugeReturn()
+
+
+class OtelMetricsForceFlushArgs(BaseModel):
+    pass
+
+
+class OtelMetricsForceFlushReturn(BaseModel):
+    success: bool = 1
```

Comment:
This should raise a typing error but we don't type check this app 💀 . We can just omit the default value. 

```suggestion
    success: bool
```

Reply:
Resolved in 773e32a2b00cb93d8499b0554139f04302c19c48